src/main/java/frc/mechs/Shooter.java

```diff
@@ -0,0 +1,77 @@
+package frc.mechs;
+
+import frc.robot.Mech;
+import frc.config.Config;
+import frc.input.Input;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+
+import edu.wpi.first.wpilibj.XboxController;
+import edu.wpi.first.wpilibj.GenericHID.Hand;
+
+public class Shooter extends Mech {
+
+    private double powerHopper;
+    private double powerFlywheel;
+    private double powerFlywheelOrig;
+
+    private final double maxSpeed;
+    private final double speedChange;
+
+    private TalonSRX motorHopper;
+    private TalonSRX motorFlywheel;
+    private XboxController controller = Input.MECH_XBOX;
+
+    private boolean shooterOn = False;
+    
+    public Shooter() {
+        this.motorHopper = new TalonSRX(Config.getInt("motor_hopper"));
+        this.motorFlywheel = new TalonSRX(Config.getInt("motor_flywheel"));
+
+        this.powerHopper = Config.getDouble("power_hopper");
+        this.powerFlywheel = Config.getDouble("power_flywheel");
+        this.powerFlywheelOrig = this.powerFlywheel;
+
+        this.maxSpeed = Config.getDouble("max_speed");
+        this.speedChange = Config.getDouble("speed_change");
+        
+        Config.defaultConfigTalon(motorHopper);
+        Config.defaultConfigTalon(motorFlywheel);
+    }
+
+    public void loop () {
+        // if x button is pressed, start shooter
+        // if y button is pressed, stop shooter
+
+        boolean xButton = controller.getXButtonReleased();
+        boolean yButton = controller.getYButtonReleased();
+
+        if (xButton && !shooterOn) shooterOn = true;
```

Comment:
I know you love the compactness of this line, but it can easily lead to buggy code. Please expand.